cpp/unreal_plugins/CoreUtils/Source/CoreUtils/ArrayDesc.h

```diff
@@ -12,7 +12,7 @@
 
 #include "CoreUtils/Rpclib.h" // MSGPACK_ADD_ENUM, MSGPACK_DEFINE_MAP
 
-// enum values must match python/spear/env.py, which is why we write them out explicitly
+// enum integer values must match python/spear/env.py, which is why we write them out explicitly
```

Comment:
Let's revert. I think this is from my dev branch but I prefer the old way.

Reply:
Done